File: 8-kyu/how-old-will-i-be-in-2099?/python/solution.py
def calculate_age(year_of_birth, current_year):
    elapsed_years = current_year - year_of_birth
    if elapsed_years > 0:
        return "You are {} year{} old.".format(elapsed_years, 's' if elapsed_years > 1 else '')
    elif elapsed_years < 0:
        return "You will be born in {} year{}.".format(-elapsed_years, 's' if -elapsed_years > 1 else '')
    else:
        return "You were born this very year!"

File: 8-kyu/how-old-will-i-be-in-2099?/c#/solution.cs
using System;

public static class AgeDiff 
{
  public static string CalculateAge(int birth, int yearTo) 
  {
    int diff = yearTo - birth;
    if (diff > 0)
      return $"You are {diff} year{(diff > 1 ? "s" :  "")} old.";
    else if (diff < 0)
      return $"You will be born in {-diff} year{(-diff > 1 ? "s" :  "")}.";
    else
      return "You were born this very year!";
  }
}